modules/lib/src/main/resources/assets/js/app/dialog/BasePublishDialog.ts

```diff
@@ -331,5 +333,13 @@ export class PublishDialogButtonRow
     setTotalInProgress(totalInProgress: number) {
         this.toggleClass('has-items-in-progress', totalInProgress > 0);
         this.getMenuActions()[0].setLabel(i18n('action.markAsReadyTotal', totalInProgress));
+        this.addAccessibilityToMarkAsReadyTotalElement();
+    }
+
+    private addAccessibilityToMarkAsReadyTotalElement(): void{
+        const menu = this.actionMenu.getChildren()
```

Comment:
1. One of the main reasons to use a typescript is to have static typing so you could always know what's behind every variable and what you're expecting to see there, that prevents from some runtime issues. So need to set types in here for all constants, inclduing those used in the callback functions.
2. However on this particular case you can access desired MenuItem by simpler code, without any type conversions:
```
private addAccessibilityToMarkAsReadyTotalElement(): void {
        const markAsReadyMenuItem: MenuItem = this.actionMenu.getMenuItem(this.getMenuActions()[0]);
        AccessibilityHelper.tabIndex(markAsReadyMenuItem);
    }
```